x/programs/rust/wasmlanche-sdk/src/program.rs

```diff
@@ -44,22 +58,22 @@ impl<K> Program<K> {
     /// Attempts to call a function `name` with `args` on the given program. This method
     /// is used to call functions on external programs.
     /// # Errors
-    /// Returns a [`StateError`] if the call fails.
+    /// Returns a [`ExternalCallError`] if the call fails.
```

Comment:
I don't think you need the "`"s 
```suggestion
    /// Returns a [ExternalCallError] if the call fails.
```
Not 100% sure though

Reply:
Yep you're right!
<img width="1035" alt="Screenshot 2024-06-11 at 19 15 11" src="https://github.com/ava-labs/hypersdk/assets/51274081/898e4eb5-9c91-4b0d-b37d-d78939f5673a">

Reply:
Sooo actually, clippy pedantic wants us to put the backticks
```sh
error: item in documentation is missing backticks
  --> x/programs/rust/wasmlanche-sdk/src/program.rs:61:20
   |
61 |     /// Returns a [ExternalCallError] if the call fails.
   |                    ^^^^^^^^^^^^^^^^^
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#doc_markdown
note: the lint level is defined here
  --> x/programs/rust/wasmlanche-sdk/src/lib.rs:1:9
   |
1  | #![deny(clippy::pedantic)]
   |         ^^^^^^^^^^^^^^^^
   = note: `#[deny(clippy::doc_markdown)]` implied by `#[deny(clippy::pedantic)]`
help: try
   |
61 |     /// Returns a [`ExternalCallError`] if the call fails.
   |                    ~~~~~~~~~~~~~~~~~~~
   ```
   
   I just added them to other parts of the code where they were missing.

Reply:
>I just added them to other parts of the code where they were missing.

Do we not have pedantic turned on everywhere?

Reply:
only on the wasmlanche-sdk